.github/workflows/testBuildHybrid.yml

```diff
@@ -1,21 +1,17 @@
-name: Build and deploy hybird apps for testing
+name: Build and deploy hybrid apps for testing
```

Comment:
I was not super involved in this conversation, but I think we should combine this with testBuild.yml and not have a specific HybridApp version. 

What's the reason for a new workflow?

Reply:
I think we should combine it but I dont think it has to be a blocker for this PR, once this works we can move it over to the testBuild workflow

Reply:
Yeah, it's easier to test new workflow when it's separated from old one but it's good idea to merge them when they are working